src/DogStatsd.php

```diff
@@ -199,20 +205,56 @@ public function updateStats($stats, $delta = 1, $sampleRate = 1.0, $tags = null)
      **/
     private function serialize_tags($tags)
     {
-        if (is_array($tags) && count($tags) > 0) {
+        $all_tags = array_merge(
+            $this->normalize_tags($tags),
+            $this->normalize_tags($this->globalTags)
+        );
+
+        if (!$all_tags) {
```

Comment:
How about using the `count` function to check the array `$all_tags` is zero. Likes:

```
if (count($all_tags) !== 0) {...}
```